### PR TITLE
Add support for PaladinPad, Arya pcb and move keyboards by KapCave into their own directory

### PR DESCRIPTION
**Pull request by nachie**
## Description

Add support for PaladinPad PCBs.

Move all keyboards maintained by nachie into the KapCave directory. This makes sure all PCBs with the same vendor ID are in the same directory. Generated a unique vendor ID (hex for KC), made sure it wasn't being used at the time.

Update the files to match the current standards required for PRs removing old code and null functions.

Add in via keymaps

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by drashna** [comment]
Just a heads up, moving keybof folders around usually means that the PR needs to target develop (you can edit the title to change the target).  If you want to still target master, then it may be worth splitting up the PR into two PRs. 

Also, there are some merge conflicts here.

**Comment by nachie** [comment]
Changed the target to develop.

Ill rebase locally to see if I can get rid of the merge conflicts

**Comment by nachie** [comment]
The lint for the new paths passes, but for some reason its trying to lint paladin64 in the old path and that is failing (since it was moved). Not sure how to handle that.

**Comment by nachie** [comment]
Resolved merged conflicts again.

**Comment by zvecr** [comment]
In its current form, this PR cannot be accepted due to the submodule changes in `lib`, most likely from before the previous develop cycle merged. These will need to be resolved before the PR can move forward.

**Comment by nachie** [comment]
> In its current form, this PR cannot be accepted due to the submodule changes in `lib`, most likely from before the previous develop cycle merged. These will need to be resolved before the PR can move forward.

Deleted the old branch, updated develop and made a new branch based off of that. This should now be rid of all the submodule changes.

File: keyboards/kapcave/gskt00/keymaps/default-poly/config.h [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#pragma once
+
+#define ENCODERS_PAD_A { D5 }
+#define ENCODERS_PAD_B { D3 }
+
+#define RGB_DI_PIN D0
+#ifdef RGB_DI_PIN
+#define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/kapcave/gskt00/keymaps/default-poly/config.h:24]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

File: keyboards/kapcave/paladinpad/rev1/config.h [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+#define DEVICE_VER      0x0001
+
+/* key matrix pins */
+#define MATRIX_ROW_PINS { C7, B6, B4, D7, D6 }
+#define MATRIX_COL_PINS { D4, D5, D3, B7 }
+
+#define RGB_DI_PIN D0
+
+#ifdef RGB_DI_PIN
+#define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/kapcave/paladinpad/rev1/config.h:30]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

File: keyboards/kapcave/paladinpad/rev2/config.h [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+/* key matrix pins */
+#define MATRIX_ROW_PINS { F4, F6, D7, B4, D6 }
+#define MATRIX_COL_PINS { F7, F5, D3, B7 }
+
+#define AUDIO_PIN C6
+
+#define RGB_DI_PIN D0
+
+#ifdef RGB_DI_PIN
+#define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/kapcave/paladinpad/rev2/config.h:32]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

File: keyboards/kapcave/paladinpad/rules.mk [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+BOOTMAGIC_ENABLE ?= yes	# Virtual DIP switch configuration(+1000)
+MOUSEKEY_ENABLE ?= no		# Mouse keys(+4700)
+EXTRAKEY_ENABLE ?= yes		# Audio control and System control(+450)
+CONSOLE_ENABLE ?= no		# Console for debug(+400)
+COMMAND_ENABLE ?= no    	# Commands for debug and configuration
+SLEEP_LED_ENABLE ?= no  	# Breathing sleep LED during USB suspend
+NKRO_ENABLE ?= yes			# USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE ?= no  	# Enable keyboard backlight functionality
+AUDIO_ENABLE ?= no			# PCB Supports this, enable if you add a buzzer
+RGBLIGHT_ENABLE ?= yes
```
Comment (drashna) [inline keyboards/kapcave/paladinpad/rules.mk:19]:
```suggestion
BOOTMAGIC_ENABLE = yes     # Enable Bootmagic Lite
MOUSEKEY_ENABLE = no       # Mouse keys
EXTRAKEY_ENABLE = yes      # Audio control and System control
CONSOLE_ENABLE = no        # Console for debug
COMMAND_ENABLE = no        # Commands for debug and configuration
SLEEP_LED_ENABLE = no      # Breathing sleep LED during USB suspend
NKRO_ENABLE = yes          # USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
AUDIO_ENABLE = no          # PCB Supports this, enable if you add a buzzer
RGBLIGHT_ENABLE = yes
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by nachie** [comment]
Thank you for the suggestions @drashna , I've accepted all. I didn't know the RGB animations could be broken out, I should be reading the core files more I guess :)

**Comment by nachie** [comment]
Adding in another PCB that is slated for GB alongside a board in the coming months.

File: keyboards/kapcave/arya/rules.mk [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+# Bootloader selection
+BOOTLOADER = stm32-dfu
+
+# Wildcard to allow APM32 MCU
+#DFU_SUFFIX_ARGS = -p FFFF -v FFFF
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Enable Bootmagic Lite
```
Comment (drashna) [inline keyboards/kapcave/arya/rules.mk:13]:
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```

File: keyboards/kapcave/arya/rules.mk [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no        # Console for debug
+COMMAND_ENABLE = no        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no       # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
```
Comment (drashna) [inline keyboards/kapcave/arya/rules.mk:24]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/kapcave/paladinpad/info.json [hunk 7]
```diff
@@ -0,0 +1,84 @@
+{
+  "keyboard_name": "PaladinPad", 
+  "url": "https://kapcave.com/products/paladinpad-pcb", 
+  "maintainer": "nachie", 
+  "width": 4, 
+  "height": 5,
```
Comment (fauxpark) [inline keyboards/kapcave/paladinpad/info.json:6]:
```suggestion
```

File: keyboards/kapcave/paladin64/rules.mk [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+COMMAND_ENABLE = no    		# Commands for debug and configuration
+SLEEP_LED_ENABLE = no  		# Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes			# USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = no  		# Enable keyboard backlight functionality
+AUDIO_ENABLE = no
+RGBLIGHT_ENABLE = yes
+# Use these options if you add in trackpoint hardware
+#PS2_MOUSE_ENABLE = yes
+#PS2_USE_USART = yes
+#PS2_USE_INT = yes
```
Comment (fauxpark) [inline keyboards/kapcave/paladin64/rules.mk:23]:
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = yes        # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
```

File: keyboards/kapcave/paladin64/readme.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
+# Paladin64 ALPS PCB
+
+Firmware for the Paladin64 ALPS64 PCB, with underglow and an optional
+trackpoint module.
+
+Keyboard Maintainer: [KapCave](https://github.com/nachie)  
+Hardware Supported: Paladin64 ALPS PCB 
+Hardware Availability: [KapCave](https://kapcave.com/products/paladin64-pcb)
```
Comment (fauxpark) [inline keyboards/kapcave/paladin64/readme.md:8]:
```suggestion
* Keyboard Maintainer: [KapCave](https://github.com/nachie)
* Hardware Supported: Paladin64 ALPS PCB
* Hardware Availability: [KapCave](https://kapcave.com/products/paladin64-pcb)
```

File: keyboards/kapcave/paladin64/info.json [hunk 10]
```diff
@@ -1,7 +1,9 @@
 {
-    "keyboard_name": "paladin64",
-    "url": "",
-    "maintainer": "qmk",
+    "keyboard_name": "Paladin64",
+    "url": "https://kapcave.com/products/paladin64-pcb",
+    "maintainer": "nachie",
+    "width": 15,
+    "height": 5,
```
Comment (fauxpark) [inline keyboards/kapcave/paladin64/info.json:6]:
```suggestion
```

File: keyboards/kapcave/gskt00/readme.md [hunk 11]
```diff
@@ -0,0 +1,23 @@
+# GSKT-00 PCB
+
+Firmware for the GSKT-00 PCB
+
+Keyboard Maintainer: [KapCave](https://github.com/nachie)  
+Hardware Supported: GSKT-00 PCB
+Hardware Availability: [KapCave](https://kapcave.com/products/gskt-00-pcb-usb-c)
```
Comment (fauxpark) [inline keyboards/kapcave/gskt00/readme.md:7]:
```suggestion
* Keyboard Maintainer: [KapCave](https://github.com/nachie)
* Hardware Supported: GSKT-00 PCB
* Hardware Availability: [KapCave](https://kapcave.com/products/gskt-00-pcb-usb-c)
```

File: keyboards/kapcave/arya/rules.mk [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+COMMAND_ENABLE = no        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no       # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+ENCODER_ENABLE = yes
+#WS2812_DRIVER = pwm
```
Comment (fauxpark) [inline keyboards/kapcave/arya/rules.mk:26]:
```suggestion
# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
ENCODER_ENABLE = yes
```

File: keyboards/kapcave/arya/readme.md [hunk 13]
```diff
@@ -0,0 +1,23 @@
+# KapCave Arya
+
+Firmware for the KapCave Arya Keyboard
+
+Keyboard Maintainer: [KapCave](https://github.com/nachie)  
+Hardware Supported: Arya PCB
+Hardware Availability: [KapCave](https://kapcave.com/products/arya-pcb)
```
Comment (fauxpark) [inline keyboards/kapcave/arya/readme.md:7]:
```suggestion
* Keyboard Maintainer: [KapCave](https://github.com/nachie)
* Hardware Supported: Arya PCB
* Hardware Availability: [KapCave](https://kapcave.com/products/arya-pcb)
```

File: keyboards/kapcave/arya/keymaps/via/keymap.c [hunk 14]
```diff
@@ -0,0 +1,63 @@
...
+                    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          \
+                    _______,          _______,          _______, _______,          _______,          _______,                            _______           \
+    ),
+	
+	[_FN3] = LAYOUT_alice_split_bs(
+        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______  , _______, _______,          \
+        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,          \
+                    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          \
+                    _______,          _______,          _______, _______,          _______,          _______,                            _______           \
```
Comment (fauxpark) [inline keyboards/kapcave/arya/keymaps/via/keymap.c:59]:
```suggestion
	[_BASE] = LAYOUT_alice_split_bs(
        KC_ESC,     KC_GESC, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_GRV,  KC_BSLS,
        KC_PGUP,    KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSPC,
        KC_PGDN,    KC_LCTL, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
                    KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, MO(1),
                    KC_LGUI,          KC_LALT,          KC_SPC,  KC_LGUI,          KC_SPC,           KC_RALT,                            KC_RCTL
    ),  

    /* FUNCTION */
	[_FN] = LAYOUT_alice_split_bs(
        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_UP  , _______, _______,
        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_LEFT, KC_RGHT,          _______,
                    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_DOWN, _______, _______,
                    _______,          _______,          _______, _______,          _______,          _______,                            _______
    ),

	[_FN2] = LAYOUT_alice_split_bs(
        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______  , _______, _______,
        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
                    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
                    _______,          _______,          _______, _______,          _______,          _______,                            _______
    ),
	
	[_FN3] = LAYOUT_alice_split_bs(
        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______  , _______, _______,
        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
                    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
                    _______,          _______,          _______, _______,          _______,          _______,                            _______
```

File: keyboards/kapcave/arya/keymaps/default/keymap.c [hunk 15]
```diff
@@ -0,0 +1,34 @@
...
+                    KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, MO(1),            \
+                    KC_LGUI,          KC_LALT,          KC_SPC,  KC_LGUI,          KC_SPC,           KC_RALT,                            KC_RCTL           \
+    ),
+    LAYOUT_alice_split_bs(
+        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_UP  , _______, _______,          \
+        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_LEFT, KC_RGHT,          _______,          \
+                    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_DOWN, _______, _______,          \
+                    _______,          _______,          _______, _______,          _______,          _______,                            _______           \
```
Comment (fauxpark) [inline keyboards/kapcave/arya/keymaps/default/keymap.c:32]:
```suggestion
    LAYOUT_alice_split_bs(
        KC_ESC,     KC_GESC, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_GRV,  KC_BSLS,
        KC_PGUP,    KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSPC,
        KC_PGDN,    KC_LCTL, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
                    KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, MO(1),
                    KC_LGUI,          KC_LALT,          KC_SPC,  KC_LGUI,          KC_SPC,           KC_RALT,                            KC_RCTL
    ),
    LAYOUT_alice_split_bs(
        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_UP  , _______, _______,
        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_LEFT, KC_RGHT,          _______,
                    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_DOWN, _______, _______,
                    _______,          _______,          _______, _______,          _______,          _______,                            _______
```

File: keyboards/kapcave/arya/info.json [hunk 16]
```diff
@@ -0,0 +1,79 @@
+{
+    "keyboard_name": "Arya", 
+    "url": "https://kapcave.com/products/arya", 
+    "maintainer": "nachie", 
+    "width": 20, 
+    "height": 14.0,
```
Comment (fauxpark) [inline keyboards/kapcave/arya/info.json:6]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/kapcave/paladinpad/paladinpad.h [hunk 17]
```diff
@@ -0,0 +1,121 @@
...
+    K_0, K_10, K_DOT1, K_ENT \
+) { \
+    { K_NUM1, K_EQ1, K_SLSH1, K_STAR1 }, \
+    { K_7, K_8, K_9, K_MINUS1 }, \
+    { K_4, K_5, K_6, K_PLUS }, \
+    { K_1, K_2, K_3, K_ENT }, \
+    { K_0, K_DOT1, K_10, K_11 } \
+}
+*/
+
```
Comment (fauxpark) [inline keyboards/kapcave/paladinpad/paladinpad.h:121]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**